src/components/PageTitle/TitlePage.js

```diff
@@ -5,6 +5,26 @@ import PropTypes from 'prop-types'
 const TitlePage = ({ title }) => (
   <Helmet>
     <title>{title}</title>
+    <meta name="title" content="" />
+    <meta name="description" content="" />
+    {/* Open Graph / Facebook */}
+    <meta property="og:type" content="website" />
+    <meta property="og:url" content="https://metatags.io/" />
+    <meta property="og:title" content="" />
+    <meta property="og:description" content="" />
+    <meta
+      property="og:image"
+      content="https://user-images.githubusercontent.com/5632966/92547334-e3e4a600-f211-11ea-864e-a4cfee7ebba8.png"
```

Comment:
this needs to be the image hosted on our site

Reply:
I'm not sure if it is posible!